### ISSUE TITLE
Spring 5.RC3 + Hibernate 5.2.10 -> error

### ISSUE THREAD

**Issue by kibertoad** [divinespear/jpa-schema-gradle-plugin#29]
When using with Spring 5 and Hibernate 5.2, I'm getting following error:

Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.springframework.instrument.classloading.SimpleThrowawayClassLoader
        at org.springframework.orm.jpa.persistenceunit.SpringPersistenceUnitInfo.getNewTempClassLoader(SpringPersistenceUnitInfo.java:90)
        at org.hibernate.jpa.boot.internal.PersistenceUnitInfoDescriptor.getTempClassLoader(PersistenceUnitInfoDescriptor.java:78)
        at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.populate(EntityManagerFactoryBuilderImpl.java:765)
        at org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.<init>(EntityManagerFactoryBuilderImpl.java:186)
        at org.hibernate.jpa.boot.spi.Bootstrap.getEntityManagerFactoryBuilder(Bootstrap.java:34)
        at org.hibernate.jpa.HibernatePersistenceProvider.getEntityManagerFactoryBuilder(HibernatePersistenceProvider.java:165)
        at org.hibernate.jpa.HibernatePersistenceProvider.getEntityManagerFactoryBuilder(HibernatePersistenceProvider.java:160)
        at org.hibernate.jpa.HibernatePersistenceProvider.generateSchema(HibernatePersistenceProvider.java:142)
        at io.github.divinespear.gradle.plugin.JpaSchemaGenerateTask.xmllessGenerate(JpaSchemaGenerateTask.groovy:429)
        at io.github.divinespear.gradle.plugin.JpaSchemaGenerateTask$_generate_closure12.doCall(JpaSchemaGenerateTask.groovy:362)
        at io.github.divinespear.gradle.plugin.JpaSchemaGenerateTask.generate(JpaSchemaGenerateTask.groovy:336)

**Comment by divinespear** [comment]
@kibertoad would you pick up your `build.gradle`?

**Comment by shenliuyang** [comment]
```
buildscript {
	dependencies{
		classpath ("org.springframework:spring-context:5.0.0.RELEASE")//fix generateSchema  get classload error
	}
}

generateSchema {
	properties = ["hibernate.dialect":dialect,
	          //fix naming_strategy conflict     "hibernate.physical_naming_strategy":"org.springframework.boot.orm.jpa.hibernate.SpringPhysicalNamingStrategy"]
}
```
@kibertoad

**Comment by divinespear** [comment]
It was about configuration change (on a long time ago...) and fixed with #31.
